app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -634,6 +618,23 @@ protected Void doInBackground(String... params) {
                     default:
                         digitalChannel.mode = EVERY_EDGE;
                 }
+
+                scienceLab.startOneChannelLA(params[0], digitalChannel.mode, params[0], 3);
+
```

Comment:
@harsh-2711 nice work :+1: Just avoid this extra line  and all okay :+1:

Reply:
@abhinavraj23 The space is to differentiate the setup with the main method of starting the Logic Analyzer. It's same in all the four modes.